contracts/samples/KnownUsersSponsor.sol

```diff
@@ -1,82 +0,0 @@
-pragma solidity ^0.5.16;
```

Comment:
why did you remove it/

Reply:
Dead code, been dead for a long time, keeps interfering with refactoring and stuff. Not worth keeping. 
Will need a separate sample repository with all of these well-maintained one day.

Reply:
sample code is in the main repository so that refactoring will require maintaining it.
putting it elsewhere means you can keep ignoring maintaining a sample
(yes, we do need to at a test to run this sample, but not having the test right now doesn't mean we need to remove it instead of refactoring it)

Reply:
We are currently ignoring the sample 100%.
The way we interact with recipients changed. The APIs changed. This is just some random solidity.
I am not maintaining it just because. The sample is NOT part of the repo. Repo should not contain unrelated code.